pom.xml

```diff
@@ -252,12 +259,6 @@
         </dependency>
 
         <dependency>
-            <groupId>commons-logging</groupId>
-            <artifactId>commons-logging</artifactId>
```

Comment:
Ok sorry, you wrote in the PR description

> It also removes an unused dependency from the pom which doesn't really warrant it's own pull request.

but actually the pom needs to be cleanup anyway and this would justifies an own PR.